frontend/public/locales/en/translation.json

```diff
@@ -2242,6 +2243,12 @@
     "tab.overview": "Overview",
     "tab.submariner": "Submariner add-ons",
     "tab.userManagement": "User management",
+    "table.acm.channel": "Hub channel",
+    "table.acm.channel.helperText": "The channel that the subscription uses to install Red Hat Advanced Cluster Management for Kubernetes.",
+    "table.acm.consoleUrl": "Hub console URL",
+    "table.acm.consoleUrl.helperText": "The console URL of Red Hat Advanced Cluster Management for Kubernetes.",
```

Comment:
Ok, sure thing; not thrilled with this sentence structure but it is not wrong so approved on my end.